oozie-to-airflow/run-sys-test

```diff
@@ -470,12 +470,15 @@ function test_composer {
 
 function test_oozie {
     header "Triggers example application on Oozie: ${LOCAL_APP_NAME}"
-    run_command "sudo oozie job --config ${REMOTE_FILESYSTEM_APP_PROPERTIES} -run"
+    JOB_ID_LINE=$(run_command "sudo oozie job --config ${REMOTE_FILESYSTEM_APP_PROPERTIES} -run" | tee /dev/tty | grep '^job: ')
+    # Skip prefix
+    # "job: " = 5 characters
+    JOB_ID=$(echo "JOB_ID_LINE" | cut -c 6-)
```

Comment:
JOB_ID_LINE misses $

Reply:
Yes. You've right. it looks like I have not sent the latest version to the remote server.

Reply:
I updated the PR.